[INFO -- 2018-12-09 03:26:13]
Starting flip_value script
version: 1.1
[INFO -- 2018-12-09 03:26:13]
Initialization strings:
[INFO -- 2018-12-09 03:26:13]
(1) file /tmp/knapsack/ga
[INFO -- 2018-12-09 03:26:13]
initSignal:1
[INFO -- 2018-12-09 03:26:13]
endSignal:8
[INFO -- 2018-12-09 03:26:13]
faultModel:4
[EXCEPTION -- 2018-12-09 03:26:17]
pythonException: empty range for randrange() (0,0, 0)
[EXCEPTION -- 2018-12-09 03:26:17]
pythonException: empty range for randrange() (0,0, 0)
[INFO -- 2018-12-09 03:26:17]
Backtrace BEGIN:
#0  __GI_raise (sig=sig@entry=6) at ../sysdeps/unix/sysv/linux/raise.c:51
#1  0x00007ffff7490f5d in __GI_abort () at abort.c:90
#2  0x00007ffff74d928d in __libc_message (action=action@entry=do_abort, fmt=fmt@entry=0x7ffff7600528 "*** Error in `%s': %s: 0x%s ***\n") at ../sysdeps/posix/libc_fatal.c:181
#3  0x00007ffff74e7650 in malloc_printerr (ar_ptr=0x0, ptr=<optimized out>, str=0x7ffff7600840 "munmap_chunk(): invalid pointer", action=<optimized out>) at malloc.c:5426
#4  munmap_chunk (p=<optimized out>) at malloc.c:2877
#5  __GI___libc_free (mem=<optimized out>) at malloc.c:3138
#6  0x0000555555559846 in __gnu_cxx::new_allocator<int>::deallocate (this=0x55555575f090 <wt2>, __p=0x5555557780a0) at /usr/include/c++/7/ext/new_allocator.h:125
#7  0x0000555555558802 in std::allocator_traits<std::allocator<int> >::deallocate (__a=..., __p=0x5555557780a0, __n=248) at /usr/include/c++/7/bits/alloc_traits.h:462
#8  0x0000555555557370 in std::_Vector_base<int, std::allocator<int> >::_M_deallocate (this=0x55555575f090 <wt2>, __p=0x5555557780a0, __n=248) at /usr/include/c++/7/bits/stl_vector.h:180
#9  0x0000555555556b87 in std::_Vector_base<int, std::allocator<int> >::~_Vector_base (this=0x55555575f090 <wt2>, __in_chrg=<optimized out>) at /usr/include/c++/7/bits/stl_vector.h:162
#10 0x0000555555555ed7 in std::vector<int, std::allocator<int> >::~vector (this=0x55555575f090 <wt2>, __in_chrg=<optimized out>) at /usr/include/c++/7/bits/stl_vector.h:435
#11 0x00007ffff7493ec0 in __run_exit_handlers (status=0, listp=0x7ffff78326f8 <__exit_funcs>, run_list_atexit=run_list_atexit@entry=true, run_dtors=run_dtors@entry=true) at exit.c:83
#12 0x00007ffff7493f1a in __GI_exit (status=<optimized out>) at exit.c:105
#13 0x0000555555554fcb in exit_gracefully () at knapsackga.cpp:26
#14 0x0000555555555165 in cost (solution=std::vector of length 200, capacity 200 = {...}) at knapsackga.cpp:48
#15 0x000055555555589e in genetic_algorithm () at knapsackga.cpp:138
#16 0x0000555555555b6c in main () at knapsackga.cpp:165

46	../sysdeps/unix/sysv/linux/raise.c: No such file or directory.

Backtrace END
Memory content before bitflip:0x310xfb0x080x00
Fault Model: LSB bit-flip
base address to flip value: 0x55555575f0b0 
address max offset: 4
Memory content after  bitflip:0x310xfb0x080x02
frame name: genetic_algorithm
symbol name: W
symbol filename: knapsackga.cpp
symbol line: 18
value: 34143025
value address: 0x55555575f0b0 <W>
Type: The type is an integer type.
Type sizeof: 4

[INFO -- 2018-12-09 03:26:17]
Fault Injection Successful after 4.8778321743011475s
[INFO -- 2018-12-09 03:26:18]
event type: exit
[EXCEPTION -- 2018-12-09 03:26:18]
exit code: no exit code available
